ublox/src/ubx_packets/packets.rs

```diff
@@ -402,6 +402,137 @@ enum NavStatusFlags2 {
     Inactive = 3,
 }
 
+/// Information message conifg
+#[ubx_packet_recv_send]
+#[ubx(
+    class = 0x06,
+    id = 0x2,
+    fixed_payload_len = 10,
+    flags = "default_for_builder"
+)]
+struct CfgInf{
+    protocol_id: u8,
+    reserved: [u8; 3],
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_0: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_1: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_2: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_3: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_4: u8,
+    #[ubx(map_type = CfgInfMask)]
+    inf_msg_mask_5: u8,
+}
+
+#[ubx_extend_bitflags]
+#[ubx(from, into_raw, rest_reserved)]
+bitflags! {
+    /// `CfgInfMask` parameters bitmask
+    #[derive(Default)]
```

Comment:
What's the default of a bitflags struct? zero? (and, if so, is zero a valid value for one of those fields?)

Reply:
It is indeed zero, and zero is a valid mask.